### PR TITLE
Add id to table rows by using table id and row id

### PR DESCRIPTION
**Pull request by ThaiWood**
This allows linking directly to table rows, please let me know if further changes are wanted.

### CODE REVIEW

File: inline-gdocs-viewer.php [hunk 1]
```diff
@@ -592,7 +592,8 @@ private function dataToHtml ($r, $options, $caption = '') {
         $id = ( 0 === $this->invocations )
             ? 'igsv-' . $this->getDocId( $options['key'] )
             : "igsv-{$this->invocations}-" . $this->getDocId( $options['key'] );
-        $html  = '<table id="' . esc_attr( $id ) . '"';
+        $id = esc_attr( $id );
+        $html  = '<table id="$id"';
```
Comment (meitar) [inline inline-gdocs-viewer.php]: Unless I am misreading this, this hunk looks equivalent to the original. Why the change? If the functional execution of this hunk is the same as the original, please leave the original in place and remove this hunk from the commit. Thanks.

**Review by fabacab — CHANGES_REQUESTED** [review]
Hey, this looks great, thank you for the patch. Please have a look at line 595 and let me know if you can adjust it as requested.

**Comment by ThaiWood** [comment]
I've reverted that section, please take another look

**Comment by fabacab** [comment]
@ThaiWood Thanks for the update. Lastly, please clean this branch. Something like this should do it, I think:

```sh
git checkout -b pr-23 fabacab/master # Make a new branch based on fabacab's "master" branch
git merge --squash add-row-id       # Prepare a squashed commit of all the commits in the local `add-row-id` branch.
git commit -m "Add id to table rows by using table id and row id" # Commit that change set.
git push origin +pr-23:add-row-id   # Forcibly overwrite the remote's ("origin") "add-row-id" branch with the contents of the new "pr-23" branch.
```

**Comment by ThaiWood** [comment]
squashed and overwritten as requested

**Comment by ThaiWood** [comment]
Fixed a typo in 616 where the quotes didn't match, and resquashed

**Comment by meitar** [comment]
This was merged but the `git svn` command rewrote the commit for compatibility with WordPress.org's Subversion server as 91f51928fd7658b3e8721604f5e6f4ae6b22863b. [Thanks and props](https://github.com/meitar/inline-gdocs-viewer/commit/4600aa71dc58121beb4590aec123d4840141b91a#diff-eb6b6c90251ab33cee784713c451e6d8R289) to your @ThaiWood user account on WordPress.org were provided in the change log and the upgrade notice. :) Many thanks for your contribution, Thai.

**Pull request closed without merging**